distribution/docker/src/docker/bin/my_init

```diff
@@ -0,0 +1,424 @@
+#!/usr/bin/python2 -u
```

Comment:
Given this is a WIP; I have a question about the design here.

Can Elasticsearch be taught to handle SIGCHLD instead of adding a new process? Or are there other benefits to having this my_init thing? It's a lot of code with a bunch of features (ssh keys, sysv-style init scripts, etc) and maybe it can be simplified?

Reply:
For now,tThis is literally a copy of what Cloud uses, but I'd like to replace it e.g. either by making ES more robust, or using something super-focussed like [tini](https://github.com/krallin/tini).